ShurikenMC/Infected-Factions
# add migrator + clean up manifest

**Summary**

this adds in the db migrator. so now db migrations are a thing that
happen. however, the migrations have to be placed inside the resources
folder instead of being read from the jar. because you can't read
folders from a jar without enumerating the jar itself. which sucks.
like major time. but for now it works :shrug:

I also cleaned up some stuff with the manifest.

**Test Plan**

- `mvn verify` works.
- Load the server the first time adding in the migrations/ dir. It runs the migrations.
- Load the server again, it doesn't try to re-run migrations.
- Look at the manifest of the newest built jar compared to before this patchset. notice how much cleaner it is, and go :open_mouth: wow
